### PR TITLE
Add `sandboxing` and `embedResources` to the builder override

### PR DESCRIPTION
**Pull request by NotAShelf**
- Formats the README (uwu)
- Adds new args to modify default pandoc behaviour

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,15 +45,21 @@ lib.evalModules {
...
 ```
-> [!NOTE]
-> `rawModules` and `evaluatedModules` are mutually exclusive.
-* `title`: the title of your documentation page
-* `templatePath`: path to a [pandoc template](https://pandoc.org/MANUAL.html#templates)
-* `styleSheetPath`: path to a Sassy CSS (SCSS) file that will compile to css
-* `codeThemePath`: path to a [pandoc syntax highlighting file](https://pandoc.org/MANUAL.html#syntax-highlighting) (note that it must be JSON with a `.theme` extension)
-* `optionsDocArgs`: additional arguments to pass to the `nixosOptionsDoc` package
+
+> [!NOTE] > `rawModules` and `evaluatedModules` are mutually exclusive.
```
Comment (eclairevoyant) [inline README.md:56]: This breaks the formatting, they need to be on separate lines

File: pkgs/builder.nix [hunk 2]
```diff
@@ -22,8 +22,14 @@
     }
   ],
   specialArgs ? {},
-  evaluatedModules ? lib.evalModules {modules = rawModules; inherit specialArgs;},
+  evaluatedModules ?
+    lib.evalModules {
+      modules = rawModules;
+      inherit specialArgs;
+    },
```
Comment (eclairevoyant) [inline pkgs/builder.nix:29]: this got addressed by f04f28063601e61019e52402a0159de7c87bf931

File: README.md [hunk 3]
```diff
@@ -8,14 +8,15 @@
...
- as well as an overlay (`overlays.default`) to allow accessing `ndg-builder` in another nixpkgs instance.
+This flake exposes a package (`packages.<system>.ndg-builder` a.k.a. `packages.<system>.default`),
+as well as an overlay (`overlays.default`) to allow accessing `ndg-builder` in another nixpkgs instance.
 
 ## Usage
 
 You can override the exposed package with the following options:
 
-* `rawModules`: a list of modules containing `options` to document. For example:
+- `rawModules`: a list of modules containing `options` to document. For example:
```
Comment (eclairevoyant) [inline README.md:18]: any reason to prefer `-` over `*`?

Comment (NotAShelf) [reply README.md:18]: Not really, it's just something my formatter does

**Comment by NotAShelf** [comment]
I self merged

**Pull request closed without merging**